packages/edit-site/src/components/global-styles/background-panel.js

```diff
@@ -31,9 +31,7 @@ export default function BackgroundPanel() {
 
 	const defaultControls = {
 		backgroundImage: true,
-		backgroundSize:
-			!! style?.background?.backgroundImage &&
-			!! inheritedStyle?.background?.backgroundImage,
+		backgroundSize: false,
```

Comment:
This means that the position controls aren't showing if a background image is set, with no size controls. Was the intention for them to be hidden? Apologies if I missed some of the evolution of the design discussion, but I thought the behaviour in global styles was intended that if you upload or add an image (or if an image is set via `theme.json`) that the size controls would always be visible.

That way when folks go to add an image, or to this part of the UI, the controls to adjust the background image are easily accessible, with the only time the controls are hidden being when there is no image.

Reply:
>  Apologies if I missed some of the evolution of the design discussion, but I thought the behaviour in global styles was intended that if you upload or add an image (or if an image is set via theme.json) that the size controls would always be visible.

That was how things started out, but it was changed to be the same UX/UI as block supports.

See: https://github.com/WordPress/gutenberg/pull/60264#issuecomment-2032396220